configure.ac

```diff
@@ -1049,11 +1049,6 @@ if test "$use_thread_local" = "yes" || test "$use_thread_local" = "auto"; then
           dnl https://gist.github.com/jamesob/fe9a872051a88b2025b1aa37bfa98605
           AC_MSG_RESULT([no])
           ;;
-        *freebsd*)
-          dnl FreeBSD's implementation of thread_local is also buggy (per
-          dnl https://groups.google.com/d/msg/bsdmailinglist/22ncTZAbDp4/Dii_pII5AwAJ)
```

Comment:
This is being removed, but from the discussion in https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=278701, [I see](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=278701#c3)
> I started investigating this from here:
https://groups.google.com/g/bsdmailinglist/c/22ncTZAbDp4/m/Dii_pII5AwAJ
> I am not sure if that comment has merit. If that is an issue, it is different than the one from this bug report.

So if it's a different issue, unrelated to the one just fixed, or it's still unclear, why is the assumption this should be removed? At least, this change seems separate from what is being done here.

Reply:
My understanding is that it is safe to use `thread_local` on FreeBSD for variables that do not have a destructor even in the presence of the above bug (or bugs if they are two separate bugs).